src/main/java/edu/kis/powp/command/ComplexCommandFactory.java

```diff
@@ -0,0 +1,36 @@
+package edu.kis.powp.command;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+public class ComplexCommandFactory {
+    private int startX;
+    private int startY;
+    private Job2dDriver driver;
+    public ComplexCommandFactory(Job2dDriver driver, int x, int y){
+        this.driver = driver;
+        this.startX = x;
+        this.startY = y;
+    }
```

Comment:
Creating a factory instance fixed on specific starting coordinates is not a very convincing idea.